projects/1.18/assets/ars-nouveau/ars_nouveau/lang/zh_cn.json

```
@@ -131,35 +142,35 @@
   "item.ars_nouveau.blank_parchment": "空白羊皮纸",
   "ars_nouveau.scribe.item_added": "物品已添加。",
   "ars_nouveau.scribe.item_removed": "物品已移除。",
-  "ars_nouveau.allow_set": "星宝石兽会只拿这些物品。",
+  "ars_nouveau.allow_set": "星宝石兽只会拿取这些物品。",
   "ars_nouveau.ignore_set": "星宝石兽会忽略这些物品。",
   "block.ars_nouveau.lava_lily": "熔岩睡莲",
   "block.ars_nouveau.volcanic_sourcelink": "火山魔源通道",
   "ars_nouveau.spell.disabled": "该法术已在配置中被禁用。",
-  "block.ars_nouveau.sourceberry_bush": "魔源浆果丛",
+  "block.ars_nouveau.sourceberry_bush": "魔源浆果",
   "ars_nouveau.connections.cleared": "连接已清除。",
-  "ars_nouveau.connections.take": "中继器已设置成从%s抽取魔力",
-  "ars_nouveau.connections.send": "中继器已设置成将魔力传输至%s",
-  "item.minecraft.splash_potion.effect.mana_regen_potion_strong": "喷溅型魔力再生药水",
-  "item.minecraft.splash_potion.effect.mana_regen_potion": "喷溅型魔力再生药水",
-  "item.minecraft.splash_potion.effect.mana_regen_potion_long": "喷溅型魔力再生药水",
-  "item.minecraft.potion.effect.mana_regen_potion_strong": "魔力再生药水",
-  "item.minecraft.potion.effect.mana_regen_potion": "魔力再生药水",
-  "item.minecraft.potion.effect.mana_regen_potion_long": "魔力再生药水",
-  "item.minecraft.lingering_potion.effect.mana_regen_potion_strong": "滞留型魔力再生药水",
-  "item.minecraft.lingering_potion.effect.mana_regen_potion": "滞留型魔力再生药水",
-  "item.minecraft.lingering_potion.effect.mana_regen_potion_long": "滞留型魔力再生药水",
-  "item.minecraft.tipped_arrow.effect.mana_regen_potion_strong": "魔力再生之箭",
-  "item.minecraft.tipped_arrow.effect.mana_regen_potion": "魔力再生之箭",
-  "item.minecraft.tipped_arrow.effect.mana_regen_potion_long": "魔力再生之箭",
-  "effect.ars_nouveau.mana_regen": "魔力再生",
+  "ars_nouveau.connections.take": "中继器将从%s抽取魔力",
+  "ars_nouveau.connections.send": "中继器会将魔力传输至%s",
+  "item.minecraft.splash_potion.effect.mana_regen_potion_strong": "喷溅型魔力恢复药水",

```

感觉原汉中的魔力再生没什么问题，毕竟原版的生命恢复也叫再生药水
```suggestion
  "item.minecraft.splash_potion.effect.mana_regen_potion_strong": "喷溅型魔力恢复药水",
```